frontend/packages/kubevirt-plugin/integration-tests/tests/cnv-uninstallation.scenario.ts

```diff
@@ -0,0 +1,26 @@
+import * as sidenavView from '@console/internal-integration-tests/views/sidenav.view';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { $, browser, by, element } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+
+const namespaceButton = $('.co-namespace-selector button');
```

Comment:
Again, should be imported from a view file

Reply:
fixed